deployer/src/config_manager.py

```diff
@@ -46,34 +57,40 @@ def __init__(self):
             self.differ = DictDiffer(self.ref_configs, self.actual_configs)
 
         def init(self):
-            output = check_output(['git', 'stash', 'list'],
-                                  cwd=self.public_dir)
+            output = helpers.check_output_decoded(['git', 'stash', 'list'],
```

Comment:
as the array is const, use a tuple
```suggestion
            output = helpers.check_output_decoded(('git', 'stash', 'list'),
```

Reply:
and you should use a const variable instead of duplicating the array everywhere

Reply:
Thanks for sharing. Will keep it in our backlog. Thanks for sharing (not the PR's topic)